### DISCUSSION TITLE
Article score, what is the ranking based on?

### DISCUSSION THREAD

**Discussion by Fennavanommen** [asreview/asreview#931] [Q&A] [answered]
Does anyone know about some sort of algorithm on which a score is based. If you mark articles as relevant or irrelavant, how does the tool select other articles? Is there a possibility to show this score and is it possible to create some sort of cut-off criterium based on this score?

**Comment by Rensvandeschoot** [comment, marked as answer]
See our paper published in [Nature Machine Intelligence](https://www.nature.com/articles/s42256-020-00287-7) "A machine learning classifier is trained to predict study relevance (labels) from a representation of the record-containing text (feature space) on the basis of prior knowledge."

Or see the explanation of Active Learning on [Read the Docs](https://asreview.readthedocs.io/en/latest/guides/activelearning.html). 

We decided not to show the outcome of the classifier (i.e, the relevance score), rather we show the rank order in the export file. This is because the relevance score itself is not calibrated and are not probability scores. Hence these values do not run between 0-1. Also, each classifier produces different values. Therefore, a cut-off criterium does not make sense. Moreover, in the case of systematic reviewing, the human is the, so-called, oracle who needs to make the final labeling decision, see also "[the Zen of ELAS](https://asreview.readthedocs.io/en/latest/intro/zen.html)".
